app/scripts/controllers/bridge/constants.ts

```diff
@@ -1,15 +1,46 @@
+import { zeroAddress } from 'ethereumjs-util';
+import { Hex } from '@metamask/utils';
+import { METABRIDGE_ETHEREUM_ADDRESS } from '../../../../shared/constants/bridge';
+import { CHAIN_IDS } from '../../../../shared/constants/network';
 import { BridgeControllerState, BridgeFeatureFlagsKey } from './types';
 
 export const BRIDGE_CONTROLLER_NAME = 'BridgeController';
+export const REFRESH_INTERVAL_MS = 30 * 1000;
+const DEFAULT_SLIPPAGE = 0.5;
+const DEFAULT_MAX_REFRESH_COUNT = 5;
+
+export enum RequestStatus {
+  LOADING,
+  FETCHED,
+  ERROR,
+}
 
 export const DEFAULT_BRIDGE_CONTROLLER_STATE: BridgeControllerState = {
   bridgeFeatureFlags: {
+    [BridgeFeatureFlagsKey.EXTENSION_CONFIG]: {
+      refreshRate: REFRESH_INTERVAL_MS,
+      maxRefreshCount: DEFAULT_MAX_REFRESH_COUNT,
+    },
     [BridgeFeatureFlagsKey.EXTENSION_SUPPORT]: false,
     [BridgeFeatureFlagsKey.NETWORK_SRC_ALLOWLIST]: [],
     [BridgeFeatureFlagsKey.NETWORK_DEST_ALLOWLIST]: [],
+    [BridgeFeatureFlagsKey.APPROVAL_GAS_MULTIPLIER]: {},
+    [BridgeFeatureFlagsKey.BRIDGE_GAS_MULTIPLIER]: {},
```

Comment:
Can we clean these up now that we don't use multipliers?

Reply:
Done in [3767cef](https://github.com/MetaMask/metamask-extension/pull/27262/commits/3767cef9728f8c02fd54980bdb5a4187b7c60224)